Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabSystemScriptingHandler.cpp

```diff
@@ -20,6 +19,31 @@
 
 namespace AzToolsFramework::Prefab
 {
+    static AZStd::unique_ptr<AZ::Entity> CreateContainerEntityAndParentEntities(const AZStd::vector<AZ::Entity*> entities)
+    {
+        bool result = false;
+        AZ::EntityId commonRoot;
+        EntityList topLevelEntities;
+        AzToolsFramework::ToolsApplicationRequestBus::BroadcastResult(
+            result, &AzToolsFramework::ToolsApplicationRequestBus::Events::FindCommonRootInactive, entities, commonRoot, &topLevelEntities);
```

Comment:
should you be checking or otherwise using this result?

Reply:
I believe the only value needed from this call is 'topLevelEntities' which is used below. 'result' and 'commonRoot' are not used and only provided because the ebus call expects them. This is preexisting code that worked fine till now. So imo, we can leave it as is.